assembl/static2/js/app/components/common/tooltips.jsx

```diff
@@ -287,20 +287,18 @@ export const textFieldToggleRequiredTooltip = (
   </Tooltip>
 );
 
-export const addPhaseTooltip = (
+export const addPhaseTooltip = () => (
```

Comment:
what does it change ?

Reply:
fieldArraywithActions wants a function as a tooltip, if it's not a function there is a bug

Reply:
🤔 ok